docs/docs/setup-vast/install/linux.md

```diff
@@ -0,0 +1,116 @@
+# Linux
+
+Use our [pre-built packages](#download) below or [build VAST from
+source](/docs/setup-vast/build-from-source) to install VAST on any Linux
+distribution. To deploy VAST as system service, you can use our [systemd
+configuration](#systemd).
+
+To package VAST...
```

Comment:
If `CPack` is available on the system you're building on you automatically get a `package` build target that packages VAST. Because this does not currently include systemd/launchd/rc.d configuration I don't think it's worth mentioning here (yet).

Reply:
Okay.